sql/src/main/java/io/crate/analyze/relations/StatementAnalysisContext.java

```diff
@@ -33,11 +34,14 @@
 
 public class StatementAnalysisContext {
 
+    public static final short MAX_RELATIONS = 256;
+
     private final Operation currentOperation;
     private final TransactionContext transactionContext;
     private final SessionContext sessionContext;
     private final Function<ParameterExpression, Symbol> convertParamFunction;
     private final List<RelationAnalysisContext> lastRelationContextQueue = new ArrayList<>();
+    private short relationCounter = 0;
```

Comment:
So that I can easily check for the MAX constraint. Otherwise eg. (byte) 240 => negative number